whois-api/src/main/java/net/ripe/db/whois/api/autocomplete/ElasticAutocompleteSearch.java

```diff
@@ -32,9 +30,11 @@
 import java.util.stream.Collectors;
 
 @Component
-@Conditional(ElasticSearchCondition.class)
 public class ElasticAutocompleteSearch implements AutocompleteSearch {
 
+    public static final String OBJECT_TYPE_FIELD_NAME = "object-type";
+    public static final String PRIMARY_KEY_FIELD_NAME = "primary-key";
+    public static final String LOOKUP_KEY_FIELD_NAME = "lookup-key";
```

Comment:
These constant should be defined in ElasticIndexService may be? As we are also using them in ElasticIndexService and there is bean injection of ElasticIndexService here or create a constant  class for them?

Reply:
I am ok moving those constant to ElasticIndexService, there are already more there